specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -8435,6 +8435,10 @@
     },
     "VirtualMachineScaleSetUpdateNetworkProfile": {
       "properties": {
+        "healthProbe": {
```

Comment:
This is a breaking change, please include this in a new api version

See 'New property added to response':
https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo

Reply:
@ryansbenson this is in the existing API version - it's just missing from the Swagger - please see the examples in the PR description

In addition it appears that link isn't publicaly available:

<img width="1022" alt="Screenshot 2019-10-01 at 10 31 10" src="https://user-images.githubusercontent.com/666005/65946502-96bf5800-e436-11e9-845a-b906a9dfdc95.png">

Reply:
@ryansbenson since this is already in the API (but is just missing from the Swagger) I'm assuming adding this optional property to an update (to match the API definition) should be fine?

Reply:
Hi @tombuildsstuff for your information, the doc above lists various types of breaking changes, which contains on entry of 
```
New property added to response 

If a new property/field is added to the response an API, the GET-PUT pipeline will be broken. Consider the case where from portal a customer updates the value of a new property "A". Another customer does a GET of this resource using the SDK. The SDK will ignore the property since it does not understand it. From the SDK, the customer does a PUT using the model that was returned from the GET. This will overwrite the change made by the first customer from the portal.  
```
Per this entry, the addition of this property is a breaking change to the existing swagger.

Reply:
Hi @ryansbenson per what @tombuildsstuff claims, this property has already existed in the API but just missing in swagger, will this condition needs a new api version?
More to ask, would proposing a new api version be done by customer?

Reply:
If the live api version accepts and supports this property, it doesn't constitute a breaking api version change (as it already exists in the api version). This is essentially a documentation issue. If these properties came after the api version was released and we're trying to document it now, your customers have already gone through the side affects of the breaking change (this isn't a good experience but not something we can solve in swagger PRs).

Approving from ARMs side